File: src/Event.java
/**
 * Клас для представлення події в системі
 */
public class Event {
    private String type;
    private Measure measure;

    public Event(String type, Measure measure) {
        this.type = type;
        this.measure = measure;
    }

    public String getType() {
        return type;
    }

    public Measure getMeasure() {
        return measure;
    }
}

File: src/Event.py
# Клас для представлення події в системі
class Event:
    def __init__(self, type_: str, measure):
        self._type = type_
        self._measure = measure

    def get_type(self) -> str:
        return self._type

    def get_measure(self):
        return self._measure
